pygpcca/__init__.py

```diff
@@ -2,5 +2,5 @@
 from pygpcca._gpcca import GPCCA, gpcca_coarsegrain
 
 __author__ = __maintainer__ = "Bernhard Reuter"
-__version__ = "1.0.3"
+__version__ = "1.0.4"
```

Comment:
@dburkhardt, please don't bump the pyGPCCA version. I will do this separately after adding some other changes.